src/loki/search.cc

```diff
@@ -579,12 +576,13 @@ struct bin_handler_t {
         // is this edge reachable in the right way
         bool reachable = reach.outbound >= p_itr->location.min_outbound_reach_ &&
                          reach.inbound >= p_itr->location.min_inbound_reach_;
-        // it's possible that it isnt reachable but the opposing is, switch to that if so
-        const GraphTile* opp_tile = tile;
         const DirectedEdge* opp_edge = nullptr;
-        if (!reachable && (opp_edge = reader.GetOpposingEdge(edge, opp_tile)) &&
-            edge_filter(opp_edge) > 0.f) {
-          auto opp_reach = check_reachability(begin, end, opp_tile, opp_edge, edge_id);
```

Comment:
the last argument here was `edge_id` but in the new code its been changed to `opp_edgeid`. without looking deeper this looks like you either fixed a bug or introduced one, seems like the former to me but i didnt check the code.